syntax = "proto3";

package myrpc;

message RpcHeader {
    bytes serviceName = 1;
    bytes methodName = 2;
    uint32 argsSize = 3;
}